server.go

```diff
@@ -68,6 +70,8 @@ type Server struct {
 
 	OpenApiSpec openapi3.T // OpenAPI spec generated by the server
 
+	Listener net.Listener
```

Comment:
You are adding a listener

So it should be closed when server stops

So please add this

```go
func (s *Server) Close () {
	if s.Listener != nil }
		s.Listener.Close()
	}
	s.Server.Close()
	return
}
```